src/main/java/domain/lotteryStore/numbers/ManualNumbersGenerator.java

```diff
@@ -0,0 +1,70 @@
+package domain.lotteryStore.numbers;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+import ui.message.ExceptionMessage;
+
+public class ManualNumbersGenerator extends LotteryNumberRange {
+    private static final String CHARACTERS_CONTAINING_NUMBER_COMMA = "^[0-9,\\s]+$";
+    private static final int SIZE_OF_NUMBERS = 6;
+    private static final String DELIMITER = ",";
+    private static final String CHARACTER_BEFORE_REPLACING = " ";
+    private static final String CHARACTER_AFTER_REPLACING = "";
+
+    private final List<Integer> manualNumbers;
+
+    public ManualNumbersGenerator(String manualNumbers) {
+        validateOtherCharacterSymbols(manualNumbers);
+        this.manualNumbers = splitWinningNumbers(manualNumbers);
+        validateSplitInput();
+    }
+
+    private void validateOtherCharacterSymbols(String input) {
+        if (!Pattern.matches(CHARACTERS_CONTAINING_NUMBER_COMMA, input)) {
+            throw new IllegalArgumentException(ExceptionMessage.INPUT_ONLY_NUMBERS_AND_COMMA.getMessage());
+        }
+    }
+
+    private List<Integer> splitWinningNumbers(String input) {
+        return Arrays.stream(input.replaceAll(CHARACTER_BEFORE_REPLACING, CHARACTER_AFTER_REPLACING)
```

Comment:
사용자를 배려해서 공백까지 제거해주셨네요 👍🏻 
```suggestion
        return Arrays.stream(input.trim()
```
trim()이란 api도 있답니다!
다양한 언어에서 이와 유사한 기능을 trim()이란 단어로 제공하니 알아두시면 좋을 것 같네요

Reply:
trim()은 맨 앞 뒤에 있는 공백만 제거해주고 중간에 있는 공백은 제거해주지 못한다고 해서 replace를 사용하였습니다